httpgrpc/README.md

```diff
@@ -0,0 +1,5 @@
+To rebuild generated protobuf code, run:
```

Comment:
I really liked the "What?" & "Why?" pattern you adopted in the cortex futures doc. It'd help a lot if the README here said what this is and why I might want to use it.